librouteros/__init__.py

```diff
@@ -57,5 +57,5 @@ def connect(host: str, username: str, password: str, **kwargs) -> typing.Type[Ap
 
 def create_transport(host: str, **kwargs) -> SocketTransport:
     sock = create_connection((host, kwargs['port']), kwargs['timeout'], (kwargs['saddr'], 0))
-    sock = kwargs['ssl_wrapper'](sock)
+    sock = kwargs['ssl_wrapper'](sock, server_hostname=host)
```

Comment:
I'd prefer to keep `ssl_wrapper` parameter as a function call with only one argument. Best would be to use:
```python3
from functools import partial

api = connect(
    username='admin',
    password='abc',
    host='some.address.com',
    ssl_wrapper=partial(ctx.wrap_socket, server_hostname='some.address.com')
    port=8729
    )
```
This code will call [wrap_socket](https://docs.python.org/3/library/ssl.html#ssl.SSLContext.wrap_socket) with `server_hostname` set.

Reply:
Yes, its worked for me. Tnx.